### PR TITLE
[plugin.audio.rne] v1.3.0+matrix.1

### PR DESCRIPTION
**Pull request by jamontes**
### Description
Hi,

This is a maintenance PR for the RNE add-on due to website changes.
I have made a Python3-only version for Matrix onwards, and decided to keep the Python2-only version for the pre-Matrix releases. I have used the "+matrix.1" suffix convention on the release version because i have seen it in other PRs to this branch.

### Changelog: 1.3.0+matrix.1 (2020.05.03)
- Code updated for matrix onwards.
- Several code fixes due to website changes.

I have summit the PR [#2883](fhttps://github.com/xbmc/repo-plugins/pull/2883) for the Helix branch in Python2-only with the maintenance changes at the same time, so the functionality is the same.

This is my first update of one of my add-ons for the Matrix branch.
Many thanks in advance for your help and best regards,
jamontes.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.rne/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+*.pyc
```
Comment (enen92) [inline plugin.audio.rne/.gitignore:1]: Please remove from submission, add to gitattributes

File: plugin.audio.rne/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+language: python
```
Comment (enen92) [inline plugin.audio.rne/.travis.yml:1]: Same

File: plugin.audio.rne/addon.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <summary lang="de_DE">Hören sie alle podcast von RNE.</summary>
+        <description lang="de_DE">Dieses add-on ermöglicht es ihnen alle podcast von der spanischen nationalen radio RNE website erneut zu hören.[CR]Im add-on einstellungs-menü, kann man verschiedene sendungen auswählen oder sich das aktuelle live streaming anhören (default).</description>
+	<summary lang="fr_FR">Ecoute tous les podcasts de RNE.</summary>
+	<description lang="fr_FR">Cette add-on permet d'écouter tous les podcasts de les émissions des programmes de la Radio National Espagnole RNE présents sur sa page web.[CR]Depuis le menu de préférences du add-on, on peut choisir entre l´option de montrer tous les programmes ou seulement ceux en émission (valeur par défault).</description>
+        <language>es</language>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```
Comment (enen92) [inline plugin.audio.rne/addon.xml:20]: Spdx

File: plugin.audio.rne/resources/tests/test_api.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```
Comment (enen92) [inline plugin.audio.rne/resources/tests/test_api.py:1]: Remove tests folder from the submission pls

File: plugin.audio.rne/.gitignore (hunk 1 above)
Comment (jamontes) [reply plugin.audio.rne/.gitignore]: OK, removed.

File: plugin.audio.rne/.travis.yml (hunk 2 above)
Comment (jamontes) [reply plugin.audio.rne/.travis.yml]: OK, no problem.

File: plugin.audio.rne/addon.xml (hunk 3 above)
Comment (jamontes) [reply plugin.audio.rne/addon.xml]: Done.

File: plugin.audio.rne/resources/tests/test_api.py (hunk 4 above)
Comment (jamontes) [reply plugin.audio.rne/resources/tests/test_api.py]: Done as well.

**Pull request merged**